newIDE/app/src/EventsSheet/ParameterFields/ObjectVariableField.js

```diff
@@ -45,17 +45,49 @@ export default React.forwardRef<ParameterFieldProps, ParameterFieldInterface>(
     });
 
     const { layout } = scope;
-    let object = null;
-    let variablesContainer = null;
-    if (objectName) {
-      if (layout && layout.hasObjectNamed(objectName)) {
-        object = layout.getObject(objectName);
-        variablesContainer = object.getVariables();
-      } else if (project && project.hasObjectNamed(objectName)) {
-        object = project.getObject(objectName);
-        variablesContainer = object.getVariables();
-      }
-    }
+    const object = objectName
+      ? layout && layout.hasObjectNamed(objectName)
+        ? layout.getObject(objectName)
+        : project && project.hasObjectNamed(objectName)
+        ? project.getObject(objectName)
+        : null
+      : null;
```

Comment:
I recommend we use this helper:
```suggestion
    const object = getObjectByName(project, layout, objectName);
```

It will a tiny bit shorter, but, more importantly, will help us to easily find everywhere in the codebase were we have this logic (which will be replaced in the future by usage of a moethod of `ObjectsContainersList` - which is an abstraction on top of two or more object containers).

Reply:
(in a sense, getObjectByName was a helper that was a hint that an abstraction was needed to handle more than 1 objects containers)